src/popup/components/Modals/SpendSuccess.vue

```diff
@@ -75,16 +85,30 @@ export default {
     resolve: { type: Function, required: true },
     transaction: { type: Object, required: true },
   },
+  data: () => ({
+    name_of_recipient: null,
+  }),
   computed: {
     ...mapState('fungibleTokens', ['availableTokens']),
     ...mapGetters(['getTxAmountTotal', 'getTxSymbol', 'getExplorerPath']),
     ...mapGetters('names', ['getPreferred']),
+    isAe() {
+      return this.getSymbol() === AETERNITY_SYMBOL;
+    },
```

Comment:
Should be done differently
https://github.com/aeternity/superhero-wallet/pull/1658